spec/fixtures/meetup_event.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
This looks like a large change but this file used to be `meetup_events` but I removed the `[]` and altered it to `meetup_event`